src/XamlStyler.Extension.Windows/source.extension.vsixmanifest

```diff
@@ -1,29 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
```

Comment:
I don't think this file shouldn't change as part of your PR

Reply:
Ok, I increased version number :)

Reply:
@grochocki Now I realized, I do not fully understand you - because of double negation :D Did you mean that file SHOULD change? And if so, how? By increasing version number?

Reply:
Oops - sorry about that! I don't think this file should change _at all_. The build pipeline handles versioning automatically.

Reply:
Ok, corrected

Reply:
I am still seeing changes. Can you revert all changes to this file?

Reply:
Oh, sorry, corrected that.